README.md

```diff
@@ -35,6 +35,20 @@ describe k8sobject(api: 'v1', type: 'pod', namespace: 'default', name: 'my-pod')
 end
 ```
 
+In order to use file resource against a file inside the pod. This is useful to identify permissions, owner, type etc..
```

Comment:
According to me, this documentation should go inside https://github.com/inspec/inspec-k8s/tree/main/docs-chef-io/content/inspec/resources